src/ringct/rctSigs.cpp

```diff
@@ -191,12 +194,11 @@ namespace rct {
             }
             else {
               Hi = hashToPoint(pk[index][i]);
-              skpkGen(alpha[i], aG[i]); //need to save alphas for later..
-              aHP[i] = scalarmultKey(Hi, alpha[i]);
+              device.mlsag_prepare(Hi, xx[i], alpha[i] , aG[i] , aHP[i] , rv.II[i]);
+              toHash[3 * i + 1] = pk[index][i];
```

Comment:
This second line seems to be a noop, right ?

Reply:
oh yes